src/server.toit

```diff
@@ -106,26 +106,33 @@ class Server:
     response_writer.write_headers STATUS_SWITCHING_PROTOCOLS
     return WebSocket response_writer.detach --no-client
 
+  // Returns true if the connection was detached, false if it was closed.
   run_connection_ connection/Connection handler/Lambda logger/log.Logger -> bool:
     if call_in_finalizer_: connection.call_in_finalizer_ = call_in_finalizer_
     while true:
       request := null
       with_timeout read_timeout:
         request = connection.read_request
-      if not request: return false
+      if not request: return false  // Timed out waiting for request from client.
```

Comment:
I would expect you to get a DEADLINE_EXCEEDED exception and not get here?

Reply:
Fixed comment, this is just if the client closes, which is normal.